core/pom.xml

```diff
@@ -103,6 +103,12 @@
 			<version>${guava.version}</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+    		<groupId>opennlp</groupId>
+    		<artifactId>maxent</artifactId>
+    		<version>3.0.0</version>
```

Comment:
The same as before. I recommend to use a recent version.

Since you are including parts of the code directly I kindly ask you to also update the NOTICE file with Apache OpenNLP attribution.